pkg/errors/errors.go

```diff
@@ -0,0 +1,106 @@
+	ErrGRPCSend = ClassNetwork.DefineError().TextualCode("ErrGRPCSend").MessageTemplate("grpc send message fail").Done()
+	// ErrStorageLoad is storage load error
+	ErrStorageLoad = ClassStorage.DefineError().TextualCode("ErrStorageLoad").MessageTemplate("load config from storage error").Done()
+	// ErrStorageSave is storage save error
+	ErrStorageSave = ClassStorage.DefineError().TextualCode("ErrStorageSave").MessageTemplate("save config to storage error").Done()
+	// ErrStorageDelete is storage delete error
+	ErrStorageDelete = ClassStorage.DefineError().TextualCode("ErrStorageDelete").MessageTemplate("delete config from storage error").Done()
+	// ErrStorageEtcdLoad is etcd storage load error
+	ErrStorageEtcdLoad = ClassStorage.DefineError().TextualCode("ErrStorageEtcdLoad").MessageTemplate("load config from storage etcd error").Done()
+	// ErrStorageEtcdSave is etcd storage save error
+	ErrStorageEtcdSave = ClassStorage.DefineError().TextualCode("ErrStorageEtcdSave").MessageTemplate("save config to storage etcd error").Done()
+	// ErrStorageEtcdDelete is etcd storage delete error
+	ErrStorageEtcdDelete = ClassStorage.DefineError().TextualCode("ErrStorageEtcdDelete").MessageTemplate("delete config from storage etcd error").Done()
+	// InternalSchedulerDuplicate is scheduler duplicate
+	InternalSchedulerDuplicate = ClassInternal.DefineError().TextualCode("InternalSchedulerDuplicate").MessageTemplate("duplicate scheduler found").Done()
+	// InternalSchedulerNotFound is scheduler not found
+	InternalSchedulerNotFound = ClassInternal.DefineError().TextualCode("InternalSchedulerNotFound").MessageTemplate("scheduler not found").Done()
+	// ErrInternalSchedulerConfig is scheduler config error
+	ErrInternalSchedulerConfig = ClassInternal.DefineError().TextualCode("ErrInternalSchedulerConfig").MessageTemplate("wrong scheduler config").Done()
+	// InternalRuleInvalid is invalid rule
+	InternalRuleInvalid = ClassInternal.DefineError().TextualCode("InternalRuleInvalid").MessageTemplate("invalid rule found").Done()
+	// InternalRuleDuplicate is duplicate rule
+	InternalRuleDuplicate = ClassInternal.DefineError().TextualCode("InternalRuleDuplicate").MessageTemplate("duplicate rule found").Done()
+	// InternalRuleMismatch is rule mismatch
+	InternalRuleMismatch = ClassInternal.DefineError().TextualCode("InternalRuleMismatch").MessageTemplate("rule key mismatch").Done()
+	// InternalOperatorNotFound is operator not found
+	InternalOperatorNotFound = ClassInternal.DefineError().TextualCode("InternalOperatorNotFound").MessageTemplate("operator not found").Done()
+	// InternalOperatorOrphan is operator orphan
+	InternalOperatorOrphan = ClassInternal.DefineError().TextualCode("InternalOperatorOrphan").MessageTemplate("operator is orphan").Done()
+	// ErrInternalOperatorMerge is operator merge error
+	ErrInternalOperatorMerge = ClassInternal.DefineError().TextualCode("ErrInternalOperatorMerge").MessageTemplate("operator cannot merge").Done()
```

Comment:
What does "operator cannot merge" mean?

Reply:
I will change it to "merge operator should be pair".